travis-infrastructure/terraform-config
# Allow cert manager webhook

## What is the problem that this PR is trying to fix?

Configure an additional firewall rule to allow the GKE control plane access to your webhook pod.
More info:
https://docs.cert-manager.io/en/latest/getting-started/webhook.html#running-on-private-gke-clusters

## What approach did you choose and why?

Add service cert-manager

## How can you test this?

Reproduce: Without rule by typing `kubectl api-resources` api returns 

> error: unable to retrieve the complete list of server APIs: admission.certmanager.k8s.io/v1beta1: the server is currently unable to handle the request

to apply rule `make plan && make apply` and verify `kubectl api-resources`

## What feedback would you like, if any?

Welcome
